opensearch_dashboards.json

```diff
@@ -16,7 +16,8 @@
     "savedObjects",
     "uiActions",
     "urlForwarding",
-    "visualizations"
+    "visualizations",
+    "management"
```

Comment:
This is to add redirection to Management datasources on the opensearch card - not sure if we should add this here or in separate PR since it seems like that may or may not happen

Reply:
removing out should be good here. As the UI is also removed for this as of now.